pkg/client/kubernetes/admissionplugins.go

```diff
@@ -80,6 +82,21 @@ func getAdmissionPluginsForVersionInternal(v string) []gardencorev1beta1.Admissi
 	return GetAdmissionPluginsForVersion(formatMajorMinor(version.Major(), version.Minor()-1))
 }
 
+// AdmissionPluginsToCommandLineParameter transforms admission plugins to a command line parameter that
+// is understood by Kubernetes components.
+func AdmissionPluginsToCommandLineParameter(ap []gardencorev1beta1.AdmissionPlugin) string {
+	if len(ap) == 0 {
+		return ""
+	}
+
+	names := make([]string, 0, len(ap))
+	for _, k := range ap {
+		names = append(names, k.Name)
+	}
+	sort.Strings(names)
```

Comment:
Why do you sort it? I guess that's not a good idea, the order is important should stay as it is.